contracts/core/Migration.sol

```diff
@@ -51,14 +58,28 @@ contract Migration is IERC721Receiver {
     }
 
     function _decreaseAllLiquidityAndCollectFees(uint256 tokenId) private returns (uint256 amount0, uint256 amount1) {
-        (uint256 amountToken0, uint256 amountToken1) = _decreaseAllLiquidity(tokenId);
+        _decreaseAllLiquidity(tokenId);
         (uint256 collectedAmount0, uint256 collectedAmount1) = _collectAllFees(tokenId);
```

Comment:
We can just change the name from `collectedAmount0 / 1`to `amount0 / 1` and remove the lines 64 and 65